src/System.Diagnostics.TraceSource/tests/TestTraceListener.cs

```diff
@@ -119,5 +119,10 @@ public override void WriteLine(string message)
         {
             Call(Method.WriteLine);
         }
+
+        public override void Close()
+        {
+            Dispose(); // In desktop Trace.Close() calls listener.Close() on every listener and in Core we call listener.Dispose() so the tests would fail in desktop without this override. 
```

Comment:
In Core when we call `Trace.Close()` we call `listener.Dispose()` and in desktop `listener.Close()`. Probably we rather dispose the objects? Because `TraceListener.Close()` is virtual, so maybe we wanted to do that so that if classes that inherit `TraceListener` don't override `Close` correctly we've taken care of disposing the object.